### DISCUSSION TITLE
Does Next11 support Preact?

### DISCUSSION THREAD

**Discussion by dross15** [vercel/next.js#28975] [Help]
I am trying to upgrade my version of next to 11.1.2, and use Preact in my next.config.js, just like here: https://github.com/vercel/next.js/blob/629884af7d3ced97b8c2ec7aebdfb1a3a5d808f0/examples/using-preact/next.config.js

But when I run build, I am seeing an error: `TypeError: Cannot read property 'framework' of undefined`

Does Next11 still support using preact?

**Comment by wwwdepot** [comment]
Use the latest preact plugin for Next.js, https://github.com/preactjs/next-plugin-preact

**Reply by dross15** [reply]
@wwwdepot when I try that - I see another error when I try to run `yarn install`: 
`➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: react-dom@npm:@preact/compat isn't supported by any available resolver`

**Reply by wwwdepot** [reply]
Looks like it doesn't support yarn berry?

**Reply by dross15** [reply]
@wwwdepot I actually got it to install, but now am seeing an error when running `yarn build`:
`ModuleNotFoundError: Module not found: Error: Can't resolve 'react' in '/Users/.../code/front-end/.yarn/__virtual__/next-virtual-728c723c55/0/cache/next-npm-11.1.2-b1c338c95c-c5a6d01b6d.zip/node_modules/next/dist/client'`
